frontend/awx/dashboard/AwxDashboard.tsx

```diff
@@ -40,12 +40,13 @@ export function AwxDashboard() {
     <PageLayout>
       {config?.ui_next && (
         <Banner variant="info">
-          <Trans>
-            <p>
-              <InfoCircleIcon /> You are currently viewing a tech preview of the new {product} user
-              interface. To return to the original interface, click <a href="/">here</a>.
-            </p>
-          </Trans>
+          <p>
+            <InfoCircleIcon />{' '}
+            <Trans>
+              You are currently viewing a tech preview of the new {{ product }} user interface. To
```

Comment:
Do we need a translation key here for the content within this Trans component to show up in the translation file?

Reply:
We do not; the `i18nKey` prop is optional. I ran `npm run i18n` and it produced the following strings in the `translation.json` file:

```
"You are currently viewing a tech preview of the new {{product}} user interface. To return to the original interface, click <3>here</3>.": "You are currently viewing a tech preview of the new {{product}} user interface. To return to the original interface, click <3>here</3>.",
```

Reply:
Great! Thanks for checking 👍